lib/plugins/aws/invokeLocal/index.js

```diff
@@ -15,14 +15,33 @@ const download = require('download');
 const mkdirp = require('mkdirp');
 const cachedir = require('cachedir');
 const jszip = require('jszip');
+const ServerlessError = require('../../../classes/Error').ServerlessError;
+const memoizee = require('memoizee');
 
 const cachePath = path.join(cachedir('serverless'), 'invokeLocal');
 
+const envVarResolvers = {
```

Comment:
@nkeating-mutualofenumclaw  it would be great if you could add kinesis resolver as well.